7/Dockerfile

```diff
@@ -1,46 +1,47 @@
-FROM centos:centos7
+FROM centos/s2i-base-centos7
 
-MAINTAINER Tom Stellard <tstellar@redhat.com>
+EXPOSE 8080
 
-ENV SUMMARY="Red Hat LLVM Toolset 7 container image" \
-    DESCRIPTION="Platform for building applications using Red Hat LLVM Toolset 7"
+ENV LLVM_VERSION="4.0.1" \
+    SUMMARY="Platform for building projects with clang/llvm" \
+    DESCRIPTION="$SUMMARY"
 
-LABEL com.redhat.component="llvm-toolset-7-docker" \
-      name="devtools/llvm-toolset-7-rhel7" \
-      version="7" \
-      summary="$SUMMARY" \
+
+LABEL summary="$SUMMARY" \
       description="$DESCRIPTION" \
       io.k8s.description="$DESCRIPTION" \
-      io.k8s.display-name="LLVM Toolset 7"
-
-RUN yum install -y yum-utils && \
-    yum install -y centos-release-scl && \
+      io.k8s.display-name="LLVM 4.0.1" \
```

Comment:
Same comment as in RHEL dockerfile, 4.0 or 4.x might be better for further maintenance.